e2e/select_test.go

```diff
@@ -402,9 +450,21 @@ func TestSelectCommand(t *testing.T) {
 				informat:      "csv",
 				compression:   true,
 				structure:     "\t",
-				outformat:     "csv",
+				outformat:     "json",
 				expectedValue: "{\"id\":\"id0\"}\n",
 			},
+			{
+				name: "input:csv,output:csv,delimiter:comma,all-versions:true,empty-bucket:true",
+				cmd: []string{
+					"select", "csv",
+					"--all-versions",
+					"--query", query,
+				},
+				informat:      "csv",
+				structure:     ",",
+				outformat:     "csv",
+				expectedValue: "-",
```

Comment:
Please refer `"-"` with a const, defined on the top of the test file and refactor other references to `"-"` be referred with the const you have defined. As a name suggestion:
```suggestion
				expectedValue: EMPTY_BUCKET_VALUE,
```
Emphasis on suggestion, a better naming is always preferred :) 

But rather than overloading `expectedValue` with a special value that implicitly skips the file uploading to the bucket, a new field to testcase such as `emptyBucket` and running the test cases accordingly might be better. This review encapsulates the change made in the test's body. Namely, this change:
```go
	if tc.expectedValue == "-" { // test empty bucket case
		src = fmt.Sprintf("s3://%s/", bucket)
		expected = ""
	} else {
		putFile(t, s3client, bucket, filename, contents)
        }
```